com.microsoft.mrtk.input/Visualizers/ControllerVisualizer/ControllerVisualizer.cs

```diff
@@ -102,7 +110,7 @@ private void RenderControllerVisuals(UnityInputSystem.InputDevice inputDevice)
                 }
                 else
                 {
-                    useFallbackVisuals = !HandsUtils.GetSubsystem().TryGetJoint(TrackedHandJoint.Palm, handNode, out _);
+                    useFallbackVisuals = HandsAggregator == null || !HandsAggregator.TryGetJoint(TrackedHandJoint.IndexTip, handNode, out _);
```

Comment:
It _should_ be an all-or-nothing hand joints report, but if you're seeing better behavior with `IndexTip` specifically, this should be fine

Reply:
Woops this should be palm, I'll change it back